pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -53,6 +54,9 @@ const (
 	DefaultCNIDir     = "/opt/cni/bin"
 
 	sysctlBridgeCallIptables = "net/bridge/bridge-nf-call-iptables"
+
+	// private mac prefix safe to use
```

Comment:
Ethernet OUI assignments http://standards-oui.ieee.org/oui.txt

Ethernet OUIs are supposed to be 6 nibbles though.